Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/components/AnimeCard.kt

```diff
@@ -0,0 +1,52 @@
+package com.abhijith.animex.ui.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.ElevatedButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import coil.compose.SubcomposeAsyncImage
+import coil.request.ImageRequest
+
+@Composable
+fun AnimeCard(name: String, modifier: Modifier = Modifier) {
+    Surface(
+        color = MaterialTheme.colorScheme.primary,
+        modifier = modifier.padding(vertical = 4.dp, horizontal = 8.dp)
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(1F),
+            verticalArrangement = Arrangement.SpaceAround,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            SubcomposeAsyncImage(
```

Comment:
SubcomposeAsyncImage vs AsyncImage?

Reply:
Looks like my understanding of SubcomposeAsyncImage is wrong. i actually wanted it to show a loading indicator based on the state which subcompose provides but from the documentation it says that it is less performant especially in cases of lists. 

changing it to AsyncImage